lib/client.js

```diff
@@ -202,6 +258,13 @@ Client.prototype._invoke = function(method, args, location, callback, options, e
     },
     xmlnsSoap = "xmlns:" + envelopeKey + "=\"http://schemas.xmlsoap.org/soap/envelope/\"";
 
+  if(this._isSequenceRequired(name)) {
```

Comment:
I had to revert to v.0.18.0 as this breaks node-soap. When a client is formed from a WSDL and it has an array inside nested object in input like this:

 ```
input: {
    ResourceId: "s:string",
    PeriodList: {
        PeriodType[]: {
             PeriodId: "s:int",
             Status: "s:string",
             DateFrom: "s:dateTime",
             DateTo: "s:dateTime"...
```

It will ignore given array in input ```PeriodType: [...]```, but will accept ```"PeriodType[]: {}"``` and it will generate malformed XML by generating ```<PeriodType[]></PeriodType[]>``` element.

v.0.18.0 Works as expected by accepting ```PeriodType: [...]```, also works when this if is disabled ```if(false && this._isSequenceRequired(name)) {```